### DISCUSSION TITLE
Search for a full list of titles

### DISCUSSION THREAD

**Discussion by elambrinaki** [SChAth/ScAthiop#1662] [Q&A] [answered]
I asked @PietroLiuzzo in #1661 how to get a list of all the titles of all the miracles in LIT2384Taamme.

@PietroLiuzzo answered my question giving the following SPARQL query:
```
SELECT ?id ?title
WHERE {?work 
    saws:formsPartOf bm:LIT2384Taamme ;
    crm:P48_has_preferred_identifier ?id ;
    dc:title ?title .}
```

which could be pasted at https://betamasaheft.eu/sparql

or if one wants to use it in another program, the same query can be sent from the URL

https://betamasaheft.eu/api/SPARQL/json?query=SELECT%20?id%20?title%20WHERE%20{?work%20saws:formsPartOf%20bm:LIT2384Taamme%20;%20crm:P48_has_preferred_identifier%20?id%20;%20dc:title%20?title%20.}

see https://betamasaheft.eu/apidoc.html

**Comment by PietroLiuzzo** [comment, marked as answer]
than it is answered, I am glad about that :)
